com.musala.simple.students.spring/src/main/java/com/musala/simple/students/spring/StudentsController.java

```diff
@@ -0,0 +1,72 @@
+package com.musala.simple.students.spring;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.validation.Valid;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.Errors;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.servlet.view.RedirectView;
+
+import com.musala.simple.students.spring.database.DatabaseType;
+import com.musala.simple.students.spring.student.Student;
+
+@Controller
+public class StudentsController {
```

Comment:
I haven't checked, but we should research if spring boot has some unit test engine we can use to write some tests here.